cloudify_types/cloudify_types/component/operations.py

```diff
@@ -124,10 +136,244 @@ def upload_blueprint(ctx, **kwargs):
     return True
 
 
+def _verify_secrets_clash(client, secrets):
```

Comment:
This should probably be called _abort_if_secrets_clash- if it was really verifying that secrets clash I'd expect it to raise an exception if they didn't.